ui/raidboss/raidemulator.html

```diff
@@ -1,88 +1,195 @@
+    <script src="emulator/overrides/RaidEmulatorTimeline.js"></script>
+    <script src="emulator/overrides/RaidEmulatorTimelineController.js"></script>
+    <script src="emulator/overrides/RaidEmulatorTimelineUI.js"></script>
+    <script src="emulator/overrides/RaidEmulatorTimelineLoader.js"></script>
+    <script src="emulator/overrides/RaidEmulatorPopupText.js"></script>
+    <script src="emulator/overrides/RaidEmulatorPopupTextGenerator.js"></script>
+    <script src="emulator/overrides/RaidEmulatorWebSocket.js"></script>
+    <script src="emulator/data/CombatantJobSearch.js"></script>
+    <script src="emulator/data/CombatantState.js"></script>
+    <script src="emulator/data/Combatant.js"></script>
+    <script src="emulator/data/CombatantTracker.js"></script>
+    <script src="emulator/EventBus.js"></script>
+    <script src="emulator/data/Encounter.js"></script>
+    <script src="emulator/data/AnalyzedEncounter.js"></script>
+    <script src="emulator/data/RaidEmulator.js"></script>
+    <script src="emulator/ui/ProgressBar.js"></script>
+    <script src="emulator/ui/EmulatedPartyInfo.js"></script>
+    <script src="emulator/ui/EmulatedMap.js"></script>
+    <script src="emulator/data/PersistorEncounter.js"></script>
+    <script src="emulator/data/Persistor.js"></script>
+    <script src="emulator/data/NetworkLogConverter.js"></script>
+    <script src="emulator/data/LogEventHandler.js"></script>
+    <script src="emulator/ui/EncounterTab.js"></script>
+    <script src="raidemulator.js"></script>
+    <link rel="stylesheet" type="text/css" href="raidemulator.css" />
+  </head>
+  <body class="bg-secondary">
+    <div class="container-fluid d-flex flex-column flex-fill h-100">
+      <div class="row my-3 progressBarRow">
+        <div class="col-auto">
+          <button
+            class="btn btn-outline-primary w-auto h-auto play"
+            type="button"
+          >
+            <i class="fa fa-play" aria-hidden="true"></i>
+          </button>
+          <button
+            class="btn btn-outline-primary w-auto h-auto pause d-none"
+            type="button"
+          >
+            <i class="fa fa-pause" aria-hidden="true"></i>
+          </button>
+        </div>
+        <div class="col">
+          <div class="progress h-100 encounterProgressBar">
+            <div
+              class="progress-bar"
+              role="progressbar"
+              aria-valuenow="0"
+              aria-valuemin="0"
+              aria-valuemax="0"
+            ></div>
+          </div>
+          <div class="playerTriggers">
+            <div class="triggerBar player0"></div>
+            <div class="triggerBar player1"></div>
+            <div class="triggerBar player2"></div>
+            <div class="triggerBar player3"></div>
+            <div class="triggerBar player4"></div>
+            <div class="triggerBar player5"></div>
+            <div class="triggerBar player6"></div>
+            <div class="triggerBar player7"></div>
```

Comment:
What about alliance raids?

Reply:
Alliance raid stuff is going to depend on the inParty tracking code. I'm also not sure how to tell alliance membership from the party log lines.

Reply:
Should I move the party tracking stuff up into v1?

Reply:
Ah no, sorry.  It's not clear to me where the v1/v2 line is.  So, maybe leave a TODO for more players and can look in the future.

Reply:
The goal would be an alliance selector above or below the party list in the UI, rather than trying to cram 24 player bars into the UI in a stack there.

Reply:
Belated thought, but ... is there actually a way to know which alliance somebody is in other than just "not in your immediate party"?